client/src/components/Buttons/ButtonBlue/ButtonBlue.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components'
+
+const ButtonBlue = ({ message }) => {
+  return <Button>{message}</Button>
```

Comment:
If you want to use this button somewhere else than in a form, I would recommend adding an `onClick` as props.

Reply:
I might add it later on.